cmd/queue/main.go

```diff
@@ -225,6 +262,13 @@ func main() {
 		zap.String(logkey.Key, servingRevisionKey),
 		zap.String(logkey.Pod, podName))
 
+	if isAction, err := handleActions(); err != nil {
+		logger.Errorw("An error occurred executing subcommand:", zap.Error(err))
```

Comment:
these two line combined are just `logger.Fatalw(...)` or Fatalf, since it seems that your logger is not sugared.

Reply:
Will update to use `logger.Fatalw`